### DISCUSSION TITLE
vscode不运行npm命令

### DISCUSSION THREAD

**Discussion by jiajia-xin** [microsoft/vscode-discussions#1453] [Extension Show and Tell]
为什么我的vscode不运行npm命令，独自打开cmd就可以运行

![62f25d1f0cc322c93d02cf28f75102b](https://github.com/user-attachments/assets/106293be-f938-49aa-a3ed-01d4d87c629d)

**Comment by robconery** [comment]
Translation for those who speak English:

> Why doesn’t my VS Code run npm commands, but it works fine when I open the command prompt separately?

**Comment by robconery** [comment]
能否请您提供更多背景信息？请尽可能添加详细内容，以便我们能够回答问题。

**Reply by robconery** [reply]
Used ChatGPT for translation hope there are no errors :).
